logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/GetOrCreateOneToOneConversationUseCase.kt

```diff
@@ -7,6 +7,7 @@ import com.wire.kalium.logic.data.conversation.ConversationGroupRepository
 import com.wire.kalium.logic.data.conversation.ConversationRepository
 import com.wire.kalium.logic.data.user.UserId
 import com.wire.kalium.logic.functional.fold
+import kotlinx.coroutines.flow.first
 
 class GetOrCreateOneToOneConversationUseCase(
```

Comment:
Let's take advantage of these changes, and we can add a small block of documentation, for the purpose of the Use case :D